index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project5 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project6 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+    </section>
+    <!--about me-->
+    <section class="aboutme">
+      <h1 class="title abtitle">About me</h1>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btstyle mainbt" type="button">Get My Resume</button>
+      <img
+        class="divisory secondline"
+        src="./icons/Line2.png"
+        alt="divisory line"
+      />
+    </section>
+    <!--Skills-->
+    <section>
+      <ul class="mainlist">
+        <li class="titles tlang">Language</li>
+        <li class="sublist">Javascript</li>
+        <li class="sublist">Ruby</li>
+        <li class="sublist">Html</li>
+        <li class="sublist">CSS</li>
+        <li class="titles tframe">Frameworks</li>
+        <li class="sublist">Bootstrap</li>
+        <li class="sublist">Ruby on Rails</li>
+        <li class="sublist">RSPec</li>
+        <li class="sublist">CapyBara</li>
+        <li class="sublist">Selenium</li>
+        <li class="titles tskill">Skills</li>
+        <li class="sublist">Codekit</li>
+        <li class="sublist">Github</li>
+        <li class="sublist">Codepen</li>
+        <li class="sublist">Gitlab</li>
+        <li class="sublist">Terminal</li>
+      </ul>
```

Comment:
- [ ] Good placing your technology categories (Language, Frameworks and Skills) in a list but the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li` tag. So under the `li` for `Language` open another `ul` list and place the technology under `Languages` there and so on

Reply:
This was very interesting, I never thought about putting a list within a list!

Reply:
Hey @Kingstalux, can you help me with this part?
I checked the list and it's supposed to be correct but Webhint Linter keeps giving me an error:
![image](https://user-images.githubusercontent.com/99830261/159130352-e3fbb573-54aa-4ac3-8ab4-ba427e430228.png)
![image](https://user-images.githubusercontent.com/99830261/159130395-b9021793-ab8d-4cc0-a92c-03b7a27bee99.png)

Reply:
On your line 217 on GitHub, you have closed the list after the h2 tag. You are to place the new list inside the li tag where the h1 tag is found

Reply:
You are to have something like this.
![Screenshot_20220319-180914_Chrome](https://user-images.githubusercontent.com/85113567/159131160-1314df66-63a5-43bd-a8ca-88ff919ef47a.jpg)